limbus/core/param.py

```diff
@@ -484,6 +487,18 @@ def disconnect(self, dst: "Param" | IterableParam) -> None:
         self._disconnect(self, dst)
 
 
+class PropParam(Param):
```

Comment:
```suggestion
class PropertyParam(Param):
```

Reply:
ok, I used Prop because was shorter, I will change it